wordpress-mobile/MediaPicker-iOS
# Update project settings to Xcode 10

This PR updates the project settings to use Xcode 10. 
It also configures Travis to use an Xcode 10 image.

To test:
 - Open the project in Xcode 10 and make sure it builds.
 - Run the sample app using one of the latest devices (iPhone XS, XR) and see if all works ok.
